Documentation/UseCases/Index.rst

```diff
@@ -272,3 +272,49 @@ like `https://example.com/downloads/detail/some-download/hz6dFgz9/`, where
 
 .. _custom dimension: https://matomo.org/docs/custom-dimensions/
 .. _setting a custom URL: https://matomo.org/faq/how-to/how-do-i-set-a-custom-url-using-the-matomo-javascript-tracker/
+
+Extending the script tag
+========================
+
+Using tracking tools like Matomo within the European Union need special treatments in order
+to let the customer consent and agree with the tracking. Although Matomo respects the browser's "Do not track" setting, not everyone is aware of it.
+
+Some GDPR tools like klaro.js require special attribute settings within the script tag in order to work.
+
+The event listener
```

Comment:
Sorry, now I get it, it's numbered, so `#.` is correct (the dot was missing the last time). But then also `.. rst-class:: bignums-xxl` must be used above.

Reply:
What did you mean with:
> But then also ` .. rst-class:: bignums-xxl` must be used above.

Checked other use cases, but only found the use below the code examples. And there only for JavaScript

Reply:
Updated the documentation according to:
* https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingReST/BignumLists.html